packages/primitives/radio/src/types.ts

```diff
@@ -0,0 +1,22 @@
+import { ReactNode, Dispatch, SetStateAction } from 'react'
+
+export type TRadioContext = [
+  string,
+  Dispatch<SetStateAction<string>>,
+]
+
+export type TRadioInput = {
+  key?: string,
+  id: string,
+  groupName: string,
+  value: string,
+  accessibilityLabel?: string,
+  accessibilityLabelBy?: string[],
+  isDisabled?: boolean,
+  onChange: (evt: any) => void,
+}
+
+export type TRadioGroup = {
+  initialValue: string,
+  children: ReactNode,
```

Comment:
can we be more specific here?
i dont know, i'm actually asking "can we?"

Reply:
Probably not, because as I showed [here](https://github.com/bubble-dev/_/pull/362#discussion_r413675987), the Group is basically a context provider, so you can put anything inside of it and wrap the RadioInputs deeper down the tree...